docs/get-started.rst

```diff
@@ -54,6 +54,16 @@ Troubleshooting
 If the install fails, you might also need to `install python-ldap's
 dependencies manually <http://stackoverflow.com/questions/4768446/python-cant-install-python-ldap>`__
 
+Docker
+~~~~~~~~~~
```

Comment:
Nit-pick - ReStructured Text actually throws warnings for underlines that don't match the length of the title. So, it should be
```
Docker
~~~~~~
```